src/topp/MzTabExporter.cpp

```diff
@@ -706,10 +710,7 @@ namespace OpenMS
         row.calc_mass_to_charge = best_ph.getCharge() != 0 ? MzTabDouble(aas.getMonoWeight(Residue::Full, best_ph.getCharge()) / best_ph.getCharge()) : MzTabDouble();
 
         // add opt_global_modified_sequence in opt_ and set it to the OpenMS amino acid string (easier human readable than unimod accessions)
-        MzTabOptionalColumnEntry opt_entry;
-        opt_entry.first = String("opt_global_modified_sequence");
-        opt_entry.second = MzTabString(aas.toString());
-        row.opt_.push_back(opt_entry);
+        row.opt_.emplace_back("opt_global_modified_sequence", aas.toString());
```

Comment:
@lukaszimmermann Can you please check the `emplace_back`? Seems to cause the build error.

Reply:
@cbielow Can I simply add a `make_pair()`?

Reply:
`emplace_back()` only considers explicit ctors. So using 
`row.opt_.emplace_back(String("opt_global_modified_sequence"), MzTabString(aas.toString()));` should work

Reply:
Lukas needs to switch on `Allow edits from maintainers.`first. Right?

Reply:
I think you should be able to push to this branch.